.prettierrc

```diff
@@ -1,3 +0,0 @@
-{
-  "proseWrap": false
```

Comment:
`"proseWrap": "never"`?

Reply:
I'm happy with the default?

Reply:
I'm fine with both `"preserve"` (default) and `"never"`, just want to make sure it's intended.

Reply:
Yeah it was intentional.